techniques/systemSettings/misc/clockConfiguration/3.1/metadata.xml

```diff
@@ -16,6 +16,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <TECHNIQUE name="Time settings">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```

Comment:
We shouldn't deprecate a version in a minor release.
This is a bug in rudder-dev